df_chat/tests/test_message_endpoint.py

```diff
@@ -17,15 +22,141 @@ def test_message_creation_success(self):
 
         message_endpoint = reverse("rooms-messages-list", kwargs={"room_pk": room.pk})
         self.client.credentials(HTTP_AUTHORIZATION="Bearer " + token)
-        response = self.client.post(message_endpoint, {"body": "Hi"})
+        body = "Test message 1234."
+        response = self.client.post(message_endpoint, {"body": body})
```

Comment:
Changed the `test_message_creation_success` test body to accommodate the following filtering tests.